server/public/model/channel_member.go

```diff
@@ -220,3 +231,8 @@ func GetDefaultChannelNotifyProps() StringMap {
 		ChannelAutoFollowThreads:        ChannelAutoFollowThreadsOff,
 	}
 }
+
+type ChannelMemberIdentifier struct {
+	ChannelId string `json:"channel_id"`
+	UserId    string `json:"user_id"`
+}
```

Comment:
Would we consider repurposing `ChannelMember` and only use the `ChannelId` and `UserId` fields for this new API instead of creating a new export?

Reply:
No, I wouldn't want to do that. I'd rather have a separate type to avoid passing unneeded data around that could be misunderstood or misused. I did discuss with Agniva passing this as a `[2]string`, but I went with the named struct to prevent devs from accidentally confusing the order of the two IDs since they're the same type and there's already related APIs that take both `(channelID, userID string)` and `(userID, channelID string)`. Also, I don't want to be the first constant length arrays to the codebase since I feel like I already made a mistake using non-pointers for `model.Preference` in a bunch of places 😛